src/ray/core_worker/task_manager.cc

```diff
@@ -792,11 +814,19 @@ void TaskManager::CompletePendingTask(const TaskID &task_id,
             reply.streaming_generator_return_ids_size();
         if (num_streaming_generator_returns > 0) {
           spec.SetNumStreamingGeneratorReturns(num_streaming_generator_returns);
-          RAY_LOG(DEBUG) << "Completed streaming generator task " << spec.TaskId()
-                         << " has " << spec.NumStreamingGeneratorReturns()
-                         << " return objects.";
+          RAY_LOG(INFO) << "Completed streaming generator task " << spec.TaskId()
+                        << " has " << spec.NumStreamingGeneratorReturns()
+                        << " return objects.";
           for (const auto &return_id_info : reply.streaming_generator_return_ids()) {
             if (return_id_info.is_plasma_object()) {
+              // NOTE(swang): It is possible that the dynamically returned refs
```

Comment:
is this a separate issue?

Reply:
Existing issue that I added a fix for in this PR.